eng/performance/helix.proj

```diff
@@ -44,7 +44,7 @@
     </HelixWorkItem>
   </ItemGroup>
   <ItemGroup Condition="!$(TargetCsproj.Contains('MicroBenchmarks.csproj')) Or '$(Architecture)' == 'arm64'">
-    <HelixWorkItem Include="WorkItem">
+    <HelixWorkItem Include="$(_BuildConfig).WorkItem">
```

Comment:
now that I look at this, can we just omit the Include on this tag? We don't reference it like we do Partition in the other branch.

Reply:
The Include is how the HelixWorkItem is named. For some reason, the ml and roslyn runs aren't showing up, though, so I need to look into this further.

Reply:
Aha. It isn't clear just from the code that this is how Helix chooses the name entirely. Would have expected `Name` to be a property on `HelixWorkItem`. Thanks for clarifying.